src/time_integration/methods_SSP.jl

```diff
@@ -280,14 +280,12 @@ function modify_dt_for_tstops!(integrator::SimpleIntegratorSSP)
             integrator.dt = integrator.tdir *
                             min(abs(integrator.dt), abs(tdir_tstop - tdir_t)) # step! to the end
         elseif iszero(integrator.dtcache) && integrator.dtchangeable
-            integrator.dt = integrator.tdir *
-                            min(abs(integrator.dt), abs(tdir_tstop - tdir_t)) # step! to the end
+            integrator.dt = integrator.tdir * min(abs(tdir_tstop - tdir_t)) # step! to the end
         elseif integrator.dtchangeable && !integrator.force_stepfail
             # always try to step! with dtcache, but lower if a tstop
             # however, if force_stepfail then don't set to dtcache, and no tstop worry
             integrator.dt = integrator.tdir *
-                            min(abs(integrator.dt), abs(integrator.dtcache),
-                                abs(tdir_tstop - tdir_t)) # step! to the end
+                            min(abs(integrator.dtcache), abs(tdir_tstop - tdir_t)) # step! to the end
```

Comment:
Please fix the comment on `dtcache` in the `struct` above

Reply:
Done in [49f3f15](https://github.com/trixi-framework/Trixi.jl/pull/1919/commits/49f3f15f8034e3933a68e3e57f64205f20085cc8)